addon/components/sl-chart.js

```diff
@@ -87,18 +110,62 @@ export default Ember.Component.extend({
         function() {
             let chartDiv = this.$( 'div.chart' );
 
+            chartDiv.highcharts( this.get( 'highchartsOptions' ) );
+            this.set( 'chart', chartDiv.highcharts() );
+            this.updateData();
+        }
+    ),
+
+    /**
+     * Updates the chart's series data
+     *
+     * @function
+     * @returns {undefined}
+     */
+    updateData: Ember.observer(
+        'series',
+        function() {
+            let chart = this.get( 'chart' );
+            let series = this.get( 'series' );
+
+            if ( !chart.hasOwnProperty( 'series' ) ) {
+                chart.series = [];
+            }
+
+            for ( let i = 0; i < series.length; i++ ) {
+                if ( chart.series.length <= i ) {
+                    chart.addSeries( series[ i ] );
+                } else {
+                    chart.series[ i ].setData( series[ i ].data );
+                }
+            }
+        }
+    ),
+
+    // -------------------------------------------------------------------------
+    // Methods
+
+    /**
+     * Options for Highcharts
+     *
+     * @function
+     * @returns {Object}
+     */
+     highchartsOptions: Ember.computed(
+        'options',
+        function() {
             let chartStyle = {
                 fontFamily: '"Benton Sans", "Helvetica Neue", Helvetica, Arial, sans-serif',
                 fontSize: '13px'
             };
 
-            let options = Ember.$.extend( true, {
-                title: '',
+            return Ember.$.extend( true, {
                 chart: {
                     animation: false,
                     backgroundColor: 'rgba(255, 255, 255, 0)',
                     style: chartStyle
                 },
+                title: null,
```

Comment:
We talked about adding a comment to this property (which is missing) that indicates why this needs to be set.  Maybe we should consider writing a test against it as well to make sure it doesn't accidentally get removed.